packages/hint/src/lib/utils/content-type.ts

```diff
@@ -1,14 +1,15 @@
 import * as fileType from 'file-type';
 import * as isSvg from 'is-svg';
-import * as mimeDB from 'mime-db';
 import { parse, MediaType } from 'content-type';
 
 import { debug as d } from './debug';
-import { IAsyncHTMLElement } from '../types';
+import { HttpHeaders, IAsyncHTMLElement } from '../types';
 import getFileExtension from './fs/file-extension';
 import getFileName from './fs/filename';
 import normalizeString from './misc/normalize-string';
 
+const mimeDB = require('mime-db');
```

Comment:
Actually this one shouldn't have been changed as it doesn't fit the same pattern (it returns an object). It was leftover from a search for better types so I'll change it back since the modern syntax is fine for this one.